### PR TITLE
load cloud provider plugins when using tkctl

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>
I' m sorry that I don't have a gcp environment. Can you make a test for this PR? @aylei :smile:

### What problem does this PR solve? 
#721
### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Helm charts change
 - Has Go code change
 - Has CI related scripts change
 - Has documents change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:
 
 ```release-note

 ```

### CODE REVIEW

**Comment by aylei** [comment]
@yeya24 IMHO, it is better to load all the auth plugins

**Comment by yeya24** [comment]
OK. Let me recheck it. Do you mean gcp, aws and alicloud? Do I need to add other plugins?

**Comment by aylei** [comment]
```
_ "k8s.io/client-go/plugin/pkg/client/auth"
```
Importing all auth plugins is sufficient I suppose

**Comment by yeya24** [comment]
Thanks for pointing out this. @aylei Updated, PTAL.

File: pkg/tkctl/cmd/cmd.go [hunk 1]
```diff
@@ -26,9 +25,12 @@ import (
 	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/list"
 	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/upinfo"
 	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/use"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/version"
 	"github.com/pingcap/tidb-operator/pkg/tkctl/config"
+
```
Comment (tennix) [inline pkg/tkctl/cmd/cmd.go]: Remove this empty line.

**Comment by yeya24** [comment]
I believe this is an upstream problem. Can we just not import azure auth plugin?

**Comment by yeya24** [comment]
Seems in the latest client-go, the code is OK. https://github.com/kubernetes/client-go/blob/4f902818859a531e018f805b2d3597f14bf2c494/plugin/pkg/client/auth/azure/azure.go#L254
``` Go
	return &azureToken{
		token: adal.Token{
			AccessToken:  accessToken,
			RefreshToken: refreshToken,
			ExpiresIn:    json.Number(expiresIn),
			ExpiresOn:    json.Number(expiresOn),
			NotBefore:    json.Number(expiresOn),
```

**Comment by yeya24** [comment]
> Seems in the latest client-go, the code is OK. https://github.com/kubernetes/client-go/blob/4f902818859a531e018f805b2d3597f14bf2c494/plugin/pkg/client/auth/azure/azure.go#L254
> 
> ```go
> 	return &azureToken{
> 		token: adal.Token{
> 			AccessToken:  accessToken,
> 			RefreshToken: refreshToken,
> 			ExpiresIn:    json.Number(expiresIn),
> 			ExpiresOn:    json.Number(expiresOn),
> 			NotBefore:    json.Number(expiresOn),
> ```

After digging into this I found out that this code was fixed into client-go v10.0.0, which is compatible for k8s 1.13+. 
If I change go.mod to client-go v10.0.0, it will produce error
``` bash
build command-line-arguments: cannot load k8s.io/api/admissionregistration/v1alpha1: cannot find module providing package k8s.io/api/admissionregistration/v1alpha1
```
And the error above can be fixed if we update k8s related package version to 1.13+. See issue https://github.com/kubernetes/client-go/issues/551

Currently we are still using 1.12.5, do we have any plan to update it recently? If not, I think it is better to not add azure auth plugin now.

**Comment by aylei** [comment]
@yeya24 I'm ok with not importing Azure auth plugin

**Comment by yeya24** [comment]
Need review for this PR.

**Comment by aylei** [comment]
@yeya24 Thanks!

**Pull request merged**